dev/extract_logs.py

```diff
@@ -0,0 +1,65 @@
+import csv
```

Comment:
Could we add a comment to the top of this file explaining what the script does and how to use it please? Similar to the format of the other python scripts in this folder. Other than that, LGTM and is probably a useful script to have in the repo.